sdcm/cluster.py

```diff
@@ -3192,6 +3192,18 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
             self.run_node_benchmarks()
         self.coredumps = {}
         super().__init__()
+        self.keyspace_info = {}
+
+    def get_keyspace_info(self, keyspace_name: str, db_node: BaseNode):
```

Comment:
let's use `@cache` on the function, instead of this property on the instance.

and then we can clear it as well with `self.get_keyspace_info.cache_clear()`

Reply:
now there's a cache, and we can remove `self.keyspace_info`

Reply:
done

Reply:
removed